content/news/2020-12-01-about-me.md

```diff
@@ -0,0 +1,111 @@
+I didn’t have to think twice about it anymore and concoct stories.
+A huge burden is lifted off my shoulders every time I tell the truth in a difficult situation, because I know it would probably be easier to lie.
+I did not lie, and the world didn’t cave in.
+
+It is because of this phase in my life that I appreciate honesty so much more.
+I appreciate it more because I know sometimes the truth can be very difficult, and it is not always easy to say. I appreciate honesty because it comes with no burden, and it leaves you free from guilt – of hurting others or causing even more damage.
+I value honesty because I have seen first hand how badly it can affect relationships with loved ones, how it can end a marriage and a life long friendship.
+Honesty is a core value for me because it has saved me from a lot of things and I can only imagine how my life would have been if I had been transparent from the start.
+
+#### Respect
+
+Respect has been a core value in my family for ages. Sometimes its easy to take what we have for granted and forget that there are people who do not have half as much. We make friends, and build relationships with people from different backgrounds with different stories. Along the way, we get to peek into other people’s worlds and see how things are done there. It can be easy to judge other people for the way they do certain things just because they do them differently than what you’re used to. And often times, this judgement can reflect in our actions and facial expressions, which might not be very respectful.
+
+I believe that part of respecting other people in society is believing that we were all created differently and there is no singular way to get anything done..
+I also believe that however learned we might be, there is always a thing or two we can learn from the people that we meet everyday.
+When we respect people, then we might be humble enough to learn from their experiences and add to ourselves one way or another.
+We all have different personalities, beliefs and values. We don’t always have to understand other people’s routines, cultures, beliefs and likes, but we can learn to accept that they are different than what our brains have accepted as the norm and that is okay.
+We are all human, we bleed the same, and we shall cease to exist some day. We all have that in common.
+
+#### Adventure
+
+Respect has been a core value in my family for ages. Sometimes its easy to take what we have for granted and forget that there are people who do not have half as much. We make friends, and build relationships with people from different backgrounds with different stories. Along the way, we get to peek into other people’s worlds and see how things are done there. It can be easy to judge other people for the way they do certain things just because they do them differently than what you’re used to. And often times, this judgement can reflect in our actions and facial expressions, which might not be very respectful.
```

Comment:
These two paragraphs are repeated from above.

Reply:
Ohh. I copied the wrong text. Thank you!